src/test/java/nl/tudelft/ewi/devhub/server/database/controllers/CommitsTest.java

```diff
@@ -101,6 +108,17 @@ public void testPersistWithParent() {
 		Commit b = createCommit(group.getRepository());
 		assertThat(b.getParents(), Matchers.contains(a));
 	}
+
+	@Test
+	public void testCommitLineChanges() {
+
+		Commit commit = createCommit(group.getRepository());
+		assertTrue(commit.getLinesAdded() == 0);
+		commit.setLinesAdded(69);
+		commit.setLinesRemoved(666);
+		assertTrue(commit.getLinesAdded() == 69);
+		assertTrue(commit.getLinesRemoved() == 666);
```

Comment:
This doesn't seem productive to me. All we are doing here is testing our setters and getters.

You should implement a test that tests if the amount of lines changed corresponds to some git diff.

Also, `assertTrue` does not produce a clear error message. Try to use `assertThat` or `assertEquals`